config.example

```diff
@@ -7,3 +7,4 @@ image_method = pixcat
 [mpd]
 host = localhost
 port = 6600
+pass = example
```

Comment:
This line should be commented out (`# pass = example`) to reflect that the default value for `pass` is no value at all.